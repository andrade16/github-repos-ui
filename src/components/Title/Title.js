import React from "react";
import "./Title.scss";

const Title = ({ title, style }) => {
  return (
    <h4 className="title" style={style}>
      {title}
    </h4>
  );
};

export default Title;
