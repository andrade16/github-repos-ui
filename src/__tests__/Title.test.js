import React from "react";
import renderer from "react-test-renderer";
import Title from "../components/Title/Title";

describe("<Title/>", () => {
  it("Renders <Title/> correctly", () => {
    const tree = renderer
      .create(<Title title="Recent Commits" style={{ margin: 0 }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
