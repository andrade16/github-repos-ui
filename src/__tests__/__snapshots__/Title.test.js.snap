// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> Renders <Title/> correctly 1`] = `
<h4
  className="title"
  style={
    Object {
      "margin": 0,
    }
  }
>
  Recent Commits
</h4>
`;
